doc/source/whatsnew/v0.18.0.txt

```diff
@@ -161,6 +161,46 @@ In addition, ``.round()`` will be available thru the ``.dt`` accessor of ``Serie
 
 .. _whatsnew_0180.api_breaking:
 
+
+Change in to_string(index=False) method of DataFrameFormatter and SeriesFormatter class
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
the underline must match the length of the stop. be a bit more concise on the title.